irony-completion.el

```diff
@@ -234,10 +221,6 @@ displayed when a derived class overrides virtual methods."
 (defun irony-completion-post-comp-placeholders (candidate)
   (cdr (nth 6 candidate)))
 
-(defun irony-completion-availability (candidate)
```

Comment:
This has been removed, but it's still used by `irony-completion--filter-candidates` below.